process-migration-service/pom.xml

```diff
@@ -399,6 +395,46 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>start-process-migration-service</id>
+                        <phase>pre-integration-test</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <executable>java</executable>
+                    <arguments>
+                        <argument>-jar</argument>
+                        <argument>${project.build.directory}/${project.build.finalName}-thorntail.jar</argument>
+                        <argument>-s./src/test/resources/project-test.yml</argument>
+                    </arguments>
+                    <async>true</async>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <configuration>
+                    <target>
+                        <sleep seconds="60" />
```

Comment:
This is not very useful and quite error-prone. Maybe we can use maven-process-plugin we already use for Spring Boot tests? It has ability to wait for URL to be reachable - it is configured with healthcheckUrl property in configuration.

Here is its configuration:

https://github.com/kiegroup/droolsjbpm-integration/blob/33ab4b666e791d7ea2d0eb901deb69231a81e990/kie-server-parent/kie-server-tests/pom.xml#L1215

Moreover, please create a wildfly profile and put everything related to integration tests under it - process-exec-maven-plugin, cargo configuration etc.

So if you leave executions of cargo and process-exec-maven-plugin under that profile, they won't be run by default, which is what we want. The reason is that PRs run with -Pwildfly enabled, but in case we want to run the integration tests from community with productized builds, we have to turn off automation in community (by not activating wildfly profile).

Thanks.

(Since this is not related to implementation, we still have time to merge this I think.)